FakeNFT/Presentation/StatisticsFlow/RatingModule/View/RatingViewController.swift

```diff
@@ -0,0 +1,198 @@
+import UIKit
+import ProgressHUD
+
+/// Контроллер  отвечает за отображение списка пользователей
+final class RatingViewController: UIViewController {
+    
+    // MARK: Private properties
+    private var viewModel: RatingViewModelProtocol
+    private lazy var refreshControl = makeRefreshControll()
+    
+    // MARK: UI
+    private lazy var ratingTableView = makeRatingTableView()
+    
+    // MARK: Initialization
+    init(viewModel: RatingViewModelProtocol) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: Override
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        viewSetup()
+        navigationItemSetup()
+        addViews()
+        activateConstraints()
+        bind()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        viewModel.checkUsers()
+        tabBarController?.tabBar.isHidden = false
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        ProgressHUD.dismiss()
+    }
+}
+
+extension RatingViewController {
```

Comment:
```suggestion
private extension RatingViewController {
```
Если все методы приватные, можно весь экстеншн сделать приватным

Reply:
Круть, спасибо!)